dev.env

```diff
@@ -62,4 +62,44 @@ export OP_ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 # In general, the command to run a deployment script will look like this:
 # forge script script/deploymentScript/DeploymentScript.s.sol --rpc-url $SEPOLIA_RPC_URL --chain-id 11155111 --private-key $WALLET_DEPLOYER_PK --etherscan-api-key $ETHERSCAN_API_KEY --verify --broadcast --legacy -vvv
 
+
```

Comment:
In general I would probably have put this into a new env file, since this is very much specific to this specific script and not very "generalized". maybe like `usp.env` or whatever. I'd probably not have this in our repo though and mainly just documented in the `workflowDeploymentAndSetupScripts/` folder. But I can be disagreed with here, especially if you (@Zitzak) say that we should just push this out like it is now and adjust later.

Reply:
Can do this. If you say documented, does that mean just to have an example version of the env I would assume? I do think that the script and env needs to be available for other deployers as well right? @marvinkruse

Reply:
yeah i probably would have added a Readme into the folder that contains a copy-pastable example for the required env file with example values. You think that makes sense? like in my mind we would have

* A `README.md` in the `workflowDeploymentAndSetupScript` folder listing the different scripts we have (currently just this one)
  * Like "hey we have these scripts for workflow deployments, see this one for queue based PIMs (link to the specific readme and script)"
* A Readme specifically for this script that is referenced in the main readme, that lists how to use it and what env is required for this, so here `QueueBasedPIM.md` or so

maybe that's overkill though, what do you think?

Reply:
yeah thats perfect fine for me! @Zitzak